tools/device-listing/device-listing.cs

```diff
@@ -35,22 +36,37 @@ static void Main(string[] args)
                 string readme = Path.Combine(directory, "README.md");
                 if (File.Exists(readme))
                 {
-                    string title = GetTitle(readme);
-                    if (title == null)
+                    var device = new DeviceInfo(readme);
+
+                    if (device.Title == null)
```

Comment:
Should we ensure the Title is unique? I suppose that will be the case in most bindings, but we might have the case where the binding differed in the second word so the Title might not be unique.

Reply:
I feel this will be easy to spot on the diff now since I started sorting. I'll leave it as is for now and add extra logic whenever we find a real case like this.